code/modules/projectiles/gun_system.dm

```diff
@@ -665,7 +665,7 @@ and you're good to go.
 //----------------------------------------------------------
 
 /obj/item/weapon/gun/proc/Fire()
-	if(QDELETED(gun_user) || !ismob(gun_user) || !able_to_fire(gun_user) || !target)
+	if(QDELETED(gun_user) || !ismob(gun_user) || !able_to_fire(gun_user) || !target || gun_user.Adjacent(target))
```

Comment:
???? This stops the gun from shooting if dude is on your side

Reply:
that's the point. That was already the case before, because you are not using Fire() to PB.

The only way you could fire on adjacent tiles was by moving towards the target, and this was a bug